web/modules/custom/dbcdk_community_devel/src/Command/GenerateCommand.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community_devel\Command\GenerateCommand.
+ */
+
+namespace Drupal\dbcdk_community_devel\Command;
+
+use DBCDK\CommunityServices\Api\CommentApi;
+use DBCDK\CommunityServices\Api\FlagApi;
+use DBCDK\CommunityServices\Api\PostApi;
+use DBCDK\CommunityServices\Api\ProfileApi;
+use DBCDK\CommunityServices\Api\QuarantineApi;
+use DBCDK\CommunityServices\ApiClient;
+use DBCDK\CommunityServices\Model\Comment;
+use DBCDK\CommunityServices\Model\Flag;
+use DBCDK\CommunityServices\Model\Post;
+use DBCDK\CommunityServices\Model\Profile;
+use DBCDK\CommunityServices\Model\Quarantine;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Drupal\Console\Command\Command;
+use Drupal\Console\Style\DrupalStyle;
+
+/**
+ * This command generates dummy content in the community service.
+ *
+ * This can be used one of more times to generate a data set for development
+ * purposes.
+ *
+ * @package Drupal\dbcdk_community_devel
+ */
+class GenerateCommand extends Command {
+
+  /**
+   * The generated profiles
+   *
+   * @var Profile[]
+   */
+  protected $profiles = [];
+
+  /**
+   * The generated quarantines.
+   *
+   * @var Quarantine[]
+   */
+  protected $quarantines = [];
+
+  /**
+   * The generated posts.
+   *
+   * @var Post[]
+   */
+  protected $posts = [];
+
+  /**
+   * The generated comments.
+   *
+   * @var Comment[]
+   */
+  protected $comments = [];
+
+  /**
+   * The generated flags.
+   *
+   * @var Flag[]
+   */
+  protected $flags = [];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function configure() {
+    $this
+      ->setName('dbcdk_community_devel:generate')
+      ->setDescription($this->trans('command.dbcdk_community_devel.generate.description'));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $io = new DrupalStyle($input, $output);
+
+    $configuration = \Drupal::service('dbcdk_community.api.configuration');
+    $api_client = new ApiClient($configuration);
```

Comment:
Why is the configuration fetched as a service, but not the client? - The same question withstands for the different API's that is also defined as services? This would eliminate the need to call the ApiClient completely :relaxed: